python_ta/checkers/forbidden_io_function_checker.py

```diff
@@ -57,16 +57,16 @@ def visit_call(self, node):
                 if (
                     isinstance(scope_parent, nodes.ClassDef)
                     and isinstance(scope, nodes.FunctionDef)
-                    and (scope_parent.name + "." + scope.name) not in self.linter.config.allowed_io
+                    and (scope_parent.name + "." + scope.name) not in self.config.allowed_io
```

Comment:
I think this is leftover from the merge conflict: you should revert the changes you see here (using `self.linter.config` rather than `self.config` throughout)